lambda/voting/backend/server.js

```diff
@@ -153,3 +192,32 @@ async function invokeLambdaFunction(requestBody) {
   console.log("\n");
   return resultJson;
 }
+
+// Utility function to read from Lambda Function through MetaKeep Lambda Read API.
+
+async function readLambdaFunction(requestBody) {
```

Comment:
how is this different from read in lambdaUtils

Reply:
removed